### ISSUE TITLE
Topo background issue

### ISSUE THREAD

**Issue by TatsuJohnson** [wvuweb/wvu-design-system#288]
The background element for wvu-bg-topo-light and wvu-bg-topo-dark uses a different version of the topographic pattern than we use in our official branding. This can be seen at https://www.wvu.edu/inclusive-campus/resources vs the background on https://campusrecreation.wvu.edu/home. 

<img width="786" alt="Screen Shot 2021-04-05 at 12 31 49 PM" src="https://user-images.githubusercontent.com/7927516/113599043-0dc94400-960c-11eb-8922-3c31be32b424.png">

**Comment by adamjohnson** [comment]
static.wvu.edu has a list of topo backgrounds in the [manifest.json file](https://static.wvu.edu/global/manifest.json).

**Comment by TatsuJohnson** [comment]
Looks like it might just be the svg version.

**Comment by zeroedin** [comment]
Yes, because the .png is large in size 375kb vs svg 93kb.  @adamglenn and I created the .svgs to reduce as much weight as possible for the homepage and I made sure to make those available on static.  And they most likely those svgs should be incorporated back into the design system.

**Comment by TatsuJohnson** [comment]
It's just that the svgs are different in design than the pngs.

**Comment by zeroedin** [comment]
It was actually created from the same source.  I believe @adamglenn cropped id differently to create the SVG.

**Comment by TatsuJohnson** [comment]
If you look at the source, I don't think the topo lines ever touch, or have sharp edges like that. Somehow the svg got simplified and messed up.
